### PR TITLE
[docu] initial version of 2.7 section in README.UPDATERS

### PR DESCRIPTION
**Pull request by M0ses**
@adrianschroeter : as discussed before - only a first draft

### CODE REVIEW

File: dist/README.UPDATERS [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+
+        zypper -n remove ruby2.1-rubygem-passenger
+
+  4. Update packages
+
+        zypper dup --no-recommends --from OBS_Server_2.7
+
+  5. Update system defaults to use new ruby version
+
+        update-alternatives --set rake /usr/bin/rake.ruby.ruby2.3
```
Comment (adrianschroeter) [inline dist/README.UPDATERS]: can we drop this and ..

File: dist/README.UPDATERS [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+
+  6. Migrate database
+
+        cd /srv/www/obs/api/
+        RAILS_ENV="production" rake db:migrate
```
Comment (adrianschroeter) [inline dist/README.UPDATERS]: use "rake.ruby2.3" here?

It is up to the user to select which rake he wants by default and rest of ruby must stay on 2.1 for yast ...

File: dist/README.UPDATERS [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+
+WARNING: Please be aware, that this section is work in progress
+
+  1. Remove the OBS 2.6 Repository
+
+        zypper rs OBS:Server:2.6
+
+  2. Add the OBS 2.7 Repository and update repository cache
+
+        zypper ar http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_13.2/OBS:Server:2.7.repo
```
Comment (adrianschroeter) [inline dist/README.UPDATERS]: better replace the openSUSE_13.2 part with somethink like "$YOUR_DISTRIBUTION" or people will add it to their installations even when they don't have a 13.2 due to copy&paste

File: dist/README.UPDATERS (hunk 1 above)
Comment (M0ses) [inline dist/README.UPDATERS]: @adrianschroeter I guess you wanted me to drop 5. only, please correct me if I`m wrong

**Pull request merged**